workflows/legiovue.nf

```diff
@@ -116,17 +128,22 @@ workflow LEGIOVUE {
             .collect{ it[1] },
         ch_quast_ref
     )
+    ch_versions = ch_versions.mix(QUAST.out.versions)
+
     SCORE_QUAST(
         QUAST.out.report
     )
+    ch_versions = ch_versions.mix(SCORE_QUAST.out.versions)
 
     // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ //
     // 6. El_Gato - Second round with assemblies for failing samples only
     // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ //
+    ch_el_gato_report = Channel.value([])
```

Comment:
I'd have to think about this more in context of the rest of the workflow, but creating an "empty" channel like this makes me wonder if there's another way of doing this?

Reply:
I'm not sure. I have it to allow someone to skip the sequence typing steps if that is not something they need. I followed the convention I saw in other pipelines to do it when the output of the steps is used later for reporting

Ex. https://github.com/nf-core/viralrecon/blob/master/workflows/nanopore.nf#L387